Library/Homebrew/dev-cmd/vendor-gems.rb

```diff
@@ -16,7 +16,7 @@ def vendor_gems_args
       EOS
 
       comma_array "--update",
-                  description: "Update all vendored Gems to the latest version."
+                  description: "Update the specified list of vendored gems to the latest version."
```

Comment:
What happens if I don't specify any arguments to `--update`?

Reply:
`Error: missing argument: --update`